src/main/java/persistence/sql/ddl/builder/CreateQueryBuilder.java

```diff
@@ -14,21 +12,13 @@ public CreateQueryBuilder(Dialect dialect) {
 
     @Override
     public String generateSQL(final Class<?> clazz) {
+        Create create = new Create(clazz, dialect);
         StringBuilder sb = new StringBuilder();
         sb.append("create table ");
-        sb.append(getTableName(clazz));
+        sb.append(create.getTableName());
         sb.append("\n(\n");
-        sb.append(getColumnSQL(clazz));
+        sb.append(create.getColumns());
         sb.append("\n);\n");
         return sb.toString();
```

Comment:
저는 개인적으로 짧은 문자열인데 빌더로 생성하면 오히려 가독성을 헤치는 것 같은 느낌을 받을때가 있어요.
개인 취향이긴한데 다른 쿼리빌더처럼 format 이나 String concat 연산으로도 충분해 보여요 😀
```suggestion
        return "create table " + create.getTableName() + "\n(\n " + create.getColumns() + " \n);\n";
```